src/protocols/utility.js

```diff
@@ -49,9 +49,9 @@ class Utility {
 	static handlePacket(packet) {
 		const methodId = packet.rmcMessage.methodId;
 
-		// Check if method is a MK7 patched method
-		if (packet.connection.accessKey === '6181dff1') {
-			UtilityMK7.handlePacket(packet);
+		// Check if game uses Storage Manager instead of Utility
+		if (packet.connection.accessKey === '6181dff1' || packet.connection.accessKey === '0fabeff2') {
+			StorageManager.handlePacket(packet);
```

Comment:
Same comment as before with legacy ranking, this should probably be a NEX version check